### PR TITLE
fix: Fixed email embed for team when specific times are chosen embeds the wrong URL

### LINKED ISSUE #14259: Email embed for team when specific times are chosen embeds the wrong URL
### Issue Summary

When trying to embed, in email, a Team's availability, the links do not include /team/ in them and therefore show a 404 when clicked through from the email. 

### Steps to Reproduce

1. Set up a Team and add availability
2. Go to team event and click the Embed button
3. Click on Email embed
4. Choose specific times to embed
5. Paste into the email 
6. When clicking on each of the selected times, they go to a page that does not include /team/ in the URL and therefore resolves to a 404. 

### Technical details

- Happened for multiple different people. 

### Evidence

Screenshot 2024-03-28 at 3 03 37 PM

Screenshot 2024-03-28 at 3 03 23 PM

### PR DESCRIPTION
**Pull request by vikrant845**
## What does this PR do?
Fixes #14259 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
1. Set up a Team and add availability
2. Go to team event and click the Embed button
3. Click on Email embed
4. Choose specific times to embed
5. Paste into the email
6. When clicking on each of the selected times, they go to the correct page.

## Mandatory Tasks

- [*] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,46 +5,6 @@ __metadata:
   version: 6
   cacheKey: 8
 
-"@0no-co/graphql.web@npm:^1.0.1":
```
Comment (CarinaWolli) [inline yarn.lock:8]: here is a guide on how to remove the changes form yarn.lock: https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
fix looks good, but yarn.lock changes need to be removed first 🙏

File: yarn.lock (hunk 1 above)
Comment (vikrant845) [reply yarn.lock:8]: I'm very sorry again. The fix didn't work for me so I had to manually delete the file, add a new commit and open a new PR.

**PR: #14948**

Please have a look at it. Thanks!!

Comment (vikrant845) [reply yarn.lock]: Done. Deleted the yarn.lock file. Please check.
**PR: #14889**

Comment (vikrant845) [reply yarn.lock]: https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

This fix wasn't working for me, the yarn.lock changes were not reverting. That's why I had to delete the yarn.lock file.

**Comment by vikrant845** [comment]
Closed due to yarn.lock issues.
Opened a new one #14975 solving the issue.

File: yarn.lock (hunk 1 above)
Comment (vikrant845) [reply yarn.lock]: Closed this one and opened a new PR #14975.

**Pull request closed without merging**